pkg/util/kubernetes/autoscalers/datadogexternal.go

```diff
@@ -184,6 +184,7 @@ func (p *Processor) updateRateLimitingMetrics() error {
 func NewDatadogClient() (*datadog.Client, error) {
 	apiKey := config.Datadog.GetString("api_key")
 	appKey := config.Datadog.GetString("app_key")
+	endpoint := config.Datadog.GetString("external_metrics_provider.endpoint")
```

Comment:
If `external_metrics_provider.endpoint ` is not specified, I think the endpoint should default to a value that's based on the configured `site` (and maybe even `dd_url`), wdyt?

Reply:
I think the idea here is: if this settings is not provided, we delegate the defaulting to the `datadog-client` lib, that will takes `DD_HOSTNAME` env var if present or use it default endpoint.

Reply:
Hmm, actually I don't think the `datadog-client` lib looks into `DD_*` vars at all, just `DATADOG_HOST`, so it would default to `https://api.datadoghq.com` whenever the new config is not specified, so we do need to write some extra code to get the defaulting right. On it 👍